src/index.html

```diff
@@ -1,17 +1,116 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div >
+          <label for="surname"> Surname:
```

Comment:
Починай текст з нового рядка
```suggestion
          <label for="surname">
             Surname:
```